### ISSUE TITLE
Fix dns site for joshandmariamusic.com

### ISSUE THREAD

**Issue by JoshuaVSherman** [WebJamApps/web-jam-back#547]
- [ ] https://joshandmariamusic.com    not working
- [ ] https://www.joshandmariamusic.com  not working

does not work with free ssl through heroku, I believe I need to purchase a ssl cert from godaddy, call them tomorrow!

**Comment by JoshuaVSherman** [comment]
@LinuxBozo Good morning Adam, hope you and Marcie are doing well. I was hoping you could provide an alternate solution for me on this issue we have where we cannot use anything other than the naked domain to redirect from. I did some research this morning and found that Godaddy does indeed suck as usual, here is the link https://www.godaddy.com/community/Managing-Domains/SSL-and-Forwarding-with-masking/td-p/114818

here is the link to the dns settings https://dcc.godaddy.com/manage/dns?domainName=JOSHANDMARIAMUSIC.COM

so basically, it seems that the redirect is working for all url variations currently EXCEPT not for https://joshandmariamusic.com

all of these seem to be forwarding just fine:
joshandmariamusic.com
www.joshandmariamusic.com
http://joshandmariamusic.com
https://www.joshandmariamusic.com

**Comment by LinuxBozo** [comment]
The reason this doesn't work for HTTPS is because the certificate issued to the site is incorrect. The certificate has these domains:
shortener.secureserver.net
www.shortener.secureserver.net

and it obviously needs to have the domains:
joshandmariamusic.com
www.joshandmariamusic.com

If you can correct the certificate then it should be fine.

**Comment by JoshuaVSherman** [comment]
@LinuxBozo Thanks for the note. I tried several things this morning and failed, also I made it worse, now www.joshandmariamusic.com is not working. I will message you after work and see if you would be available to zoom. Thanks in advance :)

**Comment by JoshuaVSherman** [comment]
@LinuxBozo figured out what I did to make it worse, it is back to just the https not working again
